integration-tests/call-builder-return-value/lib.rs

```diff
@@ -137,8 +137,7 @@ mod call_builder {
                 .submit()
                 .await
                 .expect("instantiate failed");
-            let mut call_builder_call =
-                call_builder.call_builder::<CallBuilderReturnValue>();
+            let mut contract = call_builder.call_builder::<CallBuilderReturnValue>();
```

Comment:
```suggestion
            let mut call_builder = contract.call_builder::<CallBuilderReturnValue>();
```

And change `let call_builder` on L135 to `let contract = `